packages/server-ct/src/routes-ct.js

```diff
@@ -12,7 +12,7 @@ module.exports = ({ app, config, project, onError }) => {
   app.get('/__cypress/runner/*', runnerCt.middleware(send))
 
   app.get('/__cypress/static/*', (req, res) => {
-    staticPkg.handle(req, res)
+    staticPkg.middleware(req, res)
```

Comment:
Nice, noticed that `handle` doesn't exist on this module

Reply:
Yeah, I don't know where it went 😬... I should ping someone from TR about it @flotwig or @Bkucera perhaps?

Reply:
Not familiar with this, but lol

Reply:
Looks like it exists currently? https://github.com/cypress-io/cypress/blob/a4a3c9eb7d1d6941ba71dd4deb1eb640e8a37a62/packages/server/lib/controllers/static.js#L1-L12

Reply:
https://github.com/cypress-io/cypress/blob/a4a3c9eb7d1d6941ba71dd4deb1eb640e8a37a62/packages/server/lib/routes.js#L38-L42

Reply:
It came from a bad merge. 
The `handle` function was coming from `packages/server/lib/controlloers/static`.
Which was in turn using package/static.
Now that we are using `package/static` directly, it does not make sense anymore.

Thank you @lmiller1990 for fixing it. ;)

Reply:
Yikes. I'll fix it. Thanks, that merge was super hairy.